lib/web/integrations.go

```diff
@@ -219,6 +223,79 @@ func (h *Handler) integrationsDelete(w http.ResponseWriter, r *http.Request, p h
 	return OK(), nil
 }
 
+// integrationsDelete removes an Integration based on its name
```

Comment:
```suggestion
// integrationsDeleteWithCleanup removes an Integration based on its name
```

I would update the godoc though because it does more than remove an integration.